docs/cli-guide.rst

```diff
@@ -223,76 +229,96 @@ separate elements in a list or dictionnary, but if you accidentally have
-so the system will not be able to proceed to alignment by the readalongs
+so the system will not be able to proceed to alignment by the
 aligner, SoundSwallower.
 
 If you know the language for that text, you can mark it as such in the
-XML. E.g., ``<s xml:lang="eng">This sentence is in English.</s>``. The
-``xml:lang`` attribute can be added to any element in the XML structure
+XML. E.g.:
+
+.. code-block:: xml
+
+   <s xml:lang="eng">This sentence is in English.</s>
+
+The ``xml:lang`` attribute can be added to any element in the XML structure
 and will apply to text at any depth within that element, unless the
-attribute is specified again at a deeper level, e.g.,
-``<s xml:lang="eng">English mixed with <foo xml:lang="fra">français</foo>.</s>``.
+attribute is specified again at a deeper level, e.g.:
+
+.. code-block:: xml
+
+   <s xml:lang="eng">English mixed with <foo xml:lang="fra">français</foo>.</s>
 
 There is also a simpler option available: the g2p cascade. When the g2p
 cascade is enabled, the g2p mapping will be done by first trying the
-language specified in the XML file (or with the ``-l`` flag on the
+language specified by the `xml:lang` attribute in the XML file
+(or with the first language provided to the ``-l`` flag on the
 command line, if the input is plain text). For each word where the
 result is not valid ARPABET, the g2p mapping will be attempted again
 with each of the languages specified in the g2p cascade, in order, until
-a valid ARPABET conversion is obtained. If not valid conversion is
+a valid ARPABET conversion is obtained. If no valid conversion is
 possible, are error message is printed and alignment is not attempted.
 
-To enable the g2p cascade, add the ``--g2p-fallback l1:l2:...`` option
-to ``readalongs g2p`` or ``readalongs align``:
+To enable the g2p cascade, provide multiple languages via the ``-l`` switch
+(for plain text input) or add the ``fallback-langs="l2,l3,...`` attribute to
+any element in the XML file:
 
-::
+.. code-block:: xml
+
+   <s xml:lang="eng" fallback-langs="fra,und">English mixed with français.</s>
+
+These command line examples will set the language to ``fra``, with the g2p cascade
+falling back to ``eng`` and then ``und`` when needed:
+
+.. code-block:: bash
 
-   readalongs g2p --g2p-fallback fra:eng:und myfile.tokenize.xml myfile.g2p.xml
-   readalongs align --g2p-fallback fra:eng:und myfile.xml myfile.wav output
+   readalongs prepare -l fra,eng,und myfile.txt myfile.xml
+   readalongs align -l fra,eng,und myfile.txt myfile.wav output-dir
 
 The "Undetermined" language code: und
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Notice that the two examples above use ``und`` as the last language in the
 cascade. ``und``, for Undetermined, is a special language mapping that
-uses the Unicode definition of all known characters in all alphabets, and
+uses the definition of all characters in all alphabets that are part of the
+Unicode standard, and
```

Comment:
Add sentence about the default

Reply:
rewrote parts of this paragraph to reflect the change in default behaviour.